setup.py

```diff
@@ -14,11 +14,11 @@
 requirements = galaxy_pulp_requirements + [
     "pulpcore>=3.1.1",
     "pulp-ansible>=0.2.0b8",
-    "django-prometheus>=2.0.0"
+    "django-prometheus>=2.0.0",
+    "galaxy-importer==0.2.0",
```

Comment:
`galaxy-importer` is a transitive dependency for this project. It's defined as a direct dependency in `pulp_ansible` project.
Also normally you should avoid pinning to a specific version in `setup.py` unless for a reason.